test/graphql/Offer/queries/getApprovedOffers.test.ts

```diff
@@ -33,11 +34,12 @@ const GET_APPROVED_OFFERS = gql`
 describe("getApprovedOffers", () => {
   let firstCareer: Career;
   let secondCareer: Career;
-  let approvedStudentsOffer: Offer;
-  let approvedGraduadosOffer: Offer;
-  let approvedByExtensionBothOffer: Offer;
-  let approvedByGraduadosBothOffer: Offer;
-  let approvedByGraduadosAndExtensionBothOffer: Offer;
+  let offerApprovedForStudents: Offer;
+  let offerApprovedForGraduados: Offer;
```

Comment:
las 2 primeras variables las llamaria

"offer for students approved by extension"
"offer for graduates approved by graduados"

para seguir la misma estructura de nombres